portal-api/src/v1/validation/route-validators/amendments/validate-patch-portal-facility-amendment-payload.test.ts

```diff
@@ -0,0 +1,185 @@
+import { createMocks } from 'node-mocks-http';
+import { HttpStatusCode } from 'axios';
+import { AnyObject } from '@ukef/dtfs2-common';
+import { aPortalFacilityAmendmentUserValues } from '@ukef/dtfs2-common/mock-data-backend';
+import { validatePatchPortalFacilityAmendmentPayload } from './validate-patch-portal-facility-amendment-payload';
+
+const validAmendment = JSON.parse(JSON.stringify(aPortalFacilityAmendmentUserValues())) as AnyObject;
+
+describe('validatePatchPortalFacilityAmendmentPayload', () => {
+  const invalidPayloads = [
```

Comment:
Good test coverage, can we please add few with `null` and `undefined` properties value please. Thank you